internal/daemon/daemon.go

```diff
@@ -374,13 +374,15 @@ func (d *Daemon) StartAPI(bootstrap bool, initConfig map[string]string, newConfi
 
 	// If bootstrapping the first node, just open the database and create an entry for ourselves.
 	if bootstrap {
+		internalSchema, externalSchema := d.db.Schema().Version()
 		clusterMember := cluster.InternalClusterMember{
-			Name:        localNode.Name,
-			Address:     localNode.Address.String(),
-			Certificate: localNode.Certificate.String(),
-			Schema:      d.db.Schema().Version(),
-			Heartbeat:   time.Time{},
-			Role:        cluster.Pending,
+			Name:           localNode.Name,
+			Address:        localNode.Address.String(),
+			Certificate:    localNode.Certificate.String(),
+			InternalSchema: internalSchema,
+			ExternalSchema: externalSchema,
+			Heartbeat:      time.Time{},
+			Role:           cluster.Pending,
 		}
 
```

Comment:
You could also call it here as `clusterMember.InternalSchema, clusterMember.ExternalSchema = d.db.Schema().Version()` and avoid the temp var.